jobs/rocoto/getic.sh

```diff
@@ -36,167 +36,131 @@ status=$?
 ###############################################################
 # Set script and dependency variables
 
-yyyy=$(echo $CDATE | cut -c1-4)
-mm=$(echo $CDATE | cut -c5-6)
-dd=$(echo $CDATE | cut -c7-8)
-cyc=${cyc:-$(echo $CDATE | cut -c9-10)}
+export yy=$(echo $CDATE | cut -c1-4)
+export mm=$(echo $CDATE | cut -c5-6)
+export dd=$(echo $CDATE | cut -c7-8)
+export hh=${cyc:-$(echo $CDATE | cut -c9-10)}
+export GDATE=$($NDATE -${assim_freq:-"06"} $CDATE)
+export gyy=$(echo $GDATE | cut -c1-4)
+export gmm=$(echo $GDATE | cut -c5-6)
+export gdd=$(echo $GDATE | cut -c7-8)
+export ghh=$(echo $GDATE | cut -c9-10)
+
+export DATA=${DATA:-${DATAROOT}/getic}
+export EXTRACT_DIR=${DATA:-$EXTRACT_DIR}
+export PRODHPSSDIR=${PRODHPSSDIR:-/NCEPPROD/hpssprod/runhistory}
+export COMPONENT="atmos"
+export gfs_ver=${gfs_ver:-"v16"}
+export OPS_RES=${OPS_RES:-"C768"}
+export GETICSH=${GETICSH:-${GDASINIT_DIR}/get_v16.data.sh}
+
+# Create ROTDIR/EXTRACT_DIR
+if [ ! -d $ROTDIR ]; then mkdir -p $ROTDIR ; fi
+if [ ! -d $EXTRACT_DIR ]; then mkdir -p $EXTRACT_DIR ; fi
+cd $EXTRACT_DIR
+
+# Check version, cold/warm start, and resolution
+if [[ $gfs_ver = "v16" && $EXP_WARM_START = ".true." && $CASE = $OPS_RES ]]; then # Pull warm start ICs - no chgres
+
+  # Pull RESTART files off HPSS
+  if [ ${RETRO:-"NO"} = "YES" ]; then # Retrospective parallel input
+
+    # Pull prior cycle restart files
+    htar -xvf ${HPSSDIR}/${GDATE}/gdas_restartb.tar
+    status=$?
+    [[ $status -ne 0 ]] && exit $status
 
-export COMPONENT=${COMPONENT:-atmos}
+    if [ $DO_WAVE = "YES" ]; then
```

Comment:
This only works if you are using v16 does the develop branch still support v16? If you move away from v16, this will not work. WW3 restarts/ICs are WW3 model version specific.

Reply:
I added this before we dropped wave support in develop but I kept it for possible future needs. I will remove it given your comment.

Reply:
Removed.

Reply:
@KateFriedman-NOAA it's not about supporting waves or not, it's about which version of the code/waves you are supporting.  If you are supporting the production branch, then this is necessary, but I believe you dropped support of the production branch which is what makes this unnecessary.

Reply:
Gotcha, thanks!